<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientWidget</class>
 <widget class="QWidget" name="ClientWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>329</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MultiKinect TCPIP Controller</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>TCP/IP Client</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>IP</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_IP">
        <property name="text">
         <string>0.0.0.0</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="pushButton_Disconnect">
        <property name="text">
         <string>Disconnnect</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pushButton_Connect">
        <property name="text">
         <string>Connection</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="lineEdit_Port">
        <property name="text">
         <string>0000</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="pushButton_Send">
        <property name="text">
         <string>Send Message</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="LogWidget" name="logWidget_received" native="true"/>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEdit_Message">
        <property name="text">
         <string>Hello World! :)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Port</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" rowspan="2">
       <widget class="QLabel" name="label_ConnectionState">
        <property name="text">
         <string>TCP/IP State</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="LogWidget" name="logWidget_sent" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Kinect 2 Handler</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBox_pipeline">
        <property name="currentIndex">
         <number>2</number>
        </property>
        <item>
         <property name="text">
          <string>Cpu</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>OpenGL</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>OpenCL</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Cuda</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButton_Disconnect_Devices">
        <property name="text">
         <string>Disconnect Kinects</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pushButton_Grab_Devices">
        <property name="text">
         <string>Grab All</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="TransformationWidget" name="transformationWidget_Kin2" native="true"/>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton_Connect_Devices">
        <property name="text">
         <string>Connect Kinects</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="pushButton_SendRepeated">
        <property name="text">
         <string>Continuous Grab</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="pushButton_Save_Settings">
        <property name="text">
         <string>Save Settings</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="checkBox_savePC">
        <property name="text">
         <string>Save PC</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBox_time_Resend">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="minimum">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="pushButton_Register">
        <property name="text">
         <string>Register</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="pushButton_GetPointCloud">
        <property name="text">
         <string>get Point Clouds</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="TransformationWidget" name="transformationWidget_Kin1" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_ExpertMode">
     <property name="title">
      <string>SSH Mode (Expert only)</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QWidget" name="Layout_ExpertMode" native="true">
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="1">
          <widget class="QPushButton" name="pushButton_SSHReboot">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Reboot</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="pushButton_SSHUpdate">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Update</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QLabel" name="label_State">
           <property name="text">
            <string>State</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="LogWidget" name="logWidget_ssh" native="true"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TransformationWidget</class>
   <extends>QWidget</extends>
   <header>TransformationWidget.h</header>
   <container>1</container>
   <slots>
    <signal>matrixchanged(Transform)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>LogWidget</class>
   <extends>QWidget</extends>
   <header>LogWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>groupBox_ExpertMode</sender>
   <signal>toggled(bool)</signal>
   <receiver>Layout_ExpertMode</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>103</x>
     <y>626</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>615</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
